test-resources/test-app-setup-for-revocation.json

```diff
@@ -0,0 +1,75 @@
+{
+  "_post_apps": "/* JSON body using in POST sandbox-rest.ably.io/apps request to set up the Test app */",
+  "post_apps": {
+    "limits": {
+      "presence": {
+        "maxMembers": 250
+      }
+    },
+    "keys": [
+      {
+        "revocableTokens": true
+      },
```

Comment:
Can we not just add this as a new key in `test-app-setup.json`?

Reply:
@lmars Yes of course, It's the first time using this file, so I must have thought there was a single app key - I added a separate key 591e9cd199f6c213fd4f9d4a9422283fb789c3a6 and removed file 3949a8d4b93a0c4ba5be07dcf160e3d3417cc9e4